tools/augurpy/docs/_autosummary/augurpy.estimator.rst

```diff
@@ -0,0 +1,35 @@
+﻿augurpy.estimator
```

Comment:
We don't need the augurpy docs here. We need the augurpy functions to be added to pertpy!